Explorer/Assets/DCL/InWorldCamera/PhotoDetail/PhotoDetailPoolManager.cs

```diff
@@ -0,0 +1,130 @@
+                collectionCheck: true,
+                visiblePersonDefaultCapacity,
+                visiblePersonMaxSize);
+
+            equippedWearablePool = new ObjectPool<EquippedWearableController>(
+                createFunc: () => CreateEquippedWearable(equippedWearablePrefab, webBrowser, decentralandUrlsSource, thumbnailProvider, rarityBackgrounds, rarityColors, categoryIcons),
+                actionOnGet: equippedWearable => equippedWearable.view.gameObject.SetActive(false),
+                actionOnRelease: equippedWearable => EquippedWearableRelease(equippedWearable, unusedEquippedWearablePoolObjectParent),
+                actionOnDestroy: equippedWearable => GameObject.Destroy(equippedWearable.view.gameObject),
+                collectionCheck: true,
+                equippedWearableDefaultCapacity,
+                equippedWearableMaxSize);
+
+            //Prewarm pools
+            VisiblePersonController[] visiblePersonControllers = new VisiblePersonController[visiblePersonDefaultCapacity];
+            for (int i = 0; i < visiblePersonDefaultCapacity; i++)
+                visiblePersonControllers[i] = visiblePersonPool.Get();
+            for (int i = 0; i < visiblePersonDefaultCapacity; i++)
+                visiblePersonPool.Release(visiblePersonControllers[i]);
+
+            EquippedWearableController[] equippedWearableControllers = new EquippedWearableController[equippedWearableDefaultCapacity];
+            for (int i = 0; i < equippedWearableDefaultCapacity; i++)
+                equippedWearableControllers[i] = equippedWearablePool.Get();
+            for (int i = 0; i < equippedWearableDefaultCapacity; i++)
+                equippedWearablePool.Release(equippedWearableControllers[i]);
+        }
+
+        private void EquippedWearableRelease(EquippedWearableController equippedWearable, GameObject unusedEquippedWearablePoolObjectParent)
+        {
+            equippedWearable.view.transform.SetParent(unusedEquippedWearablePoolObjectParent.transform, false);
+            equippedWearable.view.gameObject.SetActive(false);
+        }
+
+        private EquippedWearableController CreateEquippedWearable(
+            EquippedWearableView equippedWearablePrefab,
+            IWebBrowser webBrowser,
+            IDecentralandUrlsSource decentralandUrlsSource,
+            IThumbnailProvider thumbnailProvider,
+            NftTypeIconSO rarityBackgrounds,
+            NFTColorsSO rarityColors,
+            NftTypeIconSO categoryIcons)
+        {
+            EquippedWearableView view = GameObject.Instantiate(equippedWearablePrefab);
+            return new EquippedWearableController(view, webBrowser, decentralandUrlsSource, thumbnailProvider, rarityBackgrounds, rarityColors, categoryIcons);
+        }
+
+        private void VisiblePersonRelease(VisiblePersonController visiblePerson, Sprite emptyProfileImage)
+        {
+            visiblePerson.view.profileImage.SetImage(emptyProfileImage);
```

Comment:
Views should provide a method for this, instead of accessing the image directly.